src/Tooltip/Tooltip.tsx

```diff
@@ -2,13 +2,23 @@ import React from 'react';
 import Tippy from '@tippyjs/react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import 'tippy.js/dist/tippy.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'tippy.js/animations/shift-away.css';
 
 type TooltipProps = {
   children: React.ReactElement;
   content: React.ReactElement;
 };
+
+Tippy.defaultProps = {
```

Comment:
do you not put these where you have spread the other props?

Reply:
I was going to do that, but then came across this option and seems a bit pointless when they give us the option to set the defaults.